frontend/packages/console-shared/src/components/formik-fields/BaseInputField.tsx

```diff
@@ -44,6 +45,10 @@ const BaseInputField: React.FC<BaseInputFieldProps & {
           field.onChange(event);
           onChange && onChange(event);
         },
+        onBlur: (event) => {
+          field.onBlur(event);
+          onBlur && onBlur(event);
+        },
```

Comment:
Do we need to explitly provide onBlur? It's already part of props and being passed into the child component.

Reply:
I tried without passing it explicitly first. But then the Formik initial Git URL Regex validation was not running when onBlur was called, i.e. when an invalid Git URL was entered, the `Invalid Git URL` error was not showing up. 
That's why I had to explicitly define provide it. Once provided, the error message shows up as expected on receiving Invalid Git URL.